src/alerts/alerts.hpp

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ALERTS_HPP_
+#define ALERTS_HPP_
+
+#include "aos/common/cloudprotocol/alerts.hpp"
+#include "aos/common/types.hpp"
+
+namespace aos::sm::alerts {
+
+/**
+ * Service instance info.
+ */
+struct ServiceInstanceInfo {
```

Comment:
We named all internal structure as Data: ServiceData, InstanceData in contrast to ones received from cloud: ServiceInfo, InstanceInfo etc. So, rename it to `ServiceInstanceInfo`.

Reply:
fixed